Snippets/ASQFunctions/ASQFunctions_0/Usage.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage.Queue;
+using NServiceBus;
+using NServiceBus.Persistence;
+
+class Usage
+{
+    Usage(StorageQueueTriggeredEndpointConfiguration storageQueueTriggeredEndpointConfiguration, FunctionExecutionContext executionContext)
+    {
+        #region custom-diagnostics
+
+        storageQueueTriggeredEndpointConfiguration.LogDiagnostics();
+
+        #endregion
+    }
+
+    #region endpoint-configuration
+
+    static readonly FunctionEndpoint endpoint = new FunctionEndpoint(executionContext =>
+    {
+        var storageQueueTriggeredEndpointConfiguration = StorageQueueTriggeredEndpointConfiguration.FromAttributes();
+
+        return storageQueueTriggeredEndpointConfiguration;
+    });
+
+    #endregion
+
+    public static void EnablePersistence(StorageQueueTriggeredEndpointConfiguration endpointConfiguration)
+    {
+        #region enable-persistence
+
+        var persistence = endpointConfiguration.AdvancedConfiguration.UsePersistence<AzureStoragePersistence>();
+        persistence.ConnectionString("<connection-string>");
+
+        #endregion
+    }
+
+    public static void DisablePublishing(StorageQueueTriggeredEndpointConfiguration endpointConfiguration)
+    {
+        #region disable-publishing
```

Comment:
that region name is a bit confusing given the snippet-code

Reply:
Oh my, I put the wrong code in the snippet. It should be disabling the publishing. Thank you for catching it.